lighthouse-core/report/html/templates.html

```diff
@@ -45,8 +45,13 @@
 <!-- Lighthouse category header -->
 <template id="tmpl-lh-category-header">
   <div class="lh-category-header">
-    <div class="lh-score__gauge"></div>
-    <div class="lh-category-header__description"></div>
+    <div class="lh-score__gauge">
+      <!-- Will be added to .lh-gauge__label via renderCategoryHeader -->
+      <!-- <span class="tooltip-boundary">
```

Comment:
should i make this a template

Reply:
yup, i think so.  include in this one and .remove() if there's no description?